packages/core/strapi/lib/middlewares/body.js

```diff
@@ -3,12 +3,23 @@
 const fse = require('fs-extra');
 const { defaultsDeep, get } = require('lodash/fp');
 const body = require('koa-body');
+const mime = require('mime-types');
 
 const defaults = {
   multipart: true,
   patchKoa: true,
 };
 
+function ensureType(file) {
+  if (!file.type) {
+    file.type = mime.lookup(file.name);
```

Comment:
Handle the case mime.lookup returns false with `application/octet-stream`

Reply:
Added in 1e6e769c9a86982840f4fa9fda5fda729edcac3d